### PR TITLE
Fix daily build issue

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

Fix daily build issue https://github.com/ballerina-platform/ballerina-extended-library/issues/382

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: calendar/tests/test.bal [hunk 1]
```diff
@@ -429,11 +382,32 @@ function testCreateCalendar() {
     Calendar|error response = calendarClient->createCalendar(calendarMetadata);
     if (response is Calendar) {
         test:assertNotEquals(response.id.toString(), EMPTY_STRING, "Empty Calender ID.");
-        specificCalendarId = response.id.toString();
-        log:printInfo("Calendar created with ID : " +specificCalendarId);
+        calendarId = response.id.toString();
+        log:printInfo("Calendar created with ID : " +calendarId);
```
Comment (LakshanSS) [inline calendar/tests/test.bal:386]:
```suggestion
        log:printInfo("Calendar created with ID : " + calendarId);
```

File: calendar/tests/test.bal [hunk 2]
```diff
@@ -429,11 +382,32 @@ function testCreateCalendar() {
...
+}
+
+@test:AfterSuite  {}
+function afterSuiteDeleteEventandCalendar() {
+    log:printInfo("client->testDeleteCalendar()"); 
+    error? deleteResponse = calendarClient->deleteCalendar(calendarId);
+    if (deleteResponse is error) {
+        test:assertFail(msg = deleteResponse.message());
+    } else {
+        log:printInfo("Calendar deleted with ID : " +calendarId);
```
Comment (LakshanSS) [inline calendar/tests/test.bal:401]:
```suggestion
        log:printInfo("Calendar deleted with ID : " + calendarId);
```

File: calendar/tests/test.bal [hunk 3]
```diff
@@ -429,11 +382,32 @@ function testCreateCalendar() {
...
+    }
+    io:println("\n\n");
+
+    log:printInfo("client->testDeleteEvent()"); 
+    error? deleteEvent = calendarClient->deleteEvent(eventId);
+    if (deleteEvent is error) {
+        test:assertFail(msg = deleteEvent.message());
+    } else {
+        log:printInfo("Event deleted with ID : " +eventId);
```
Comment (LakshanSS) [inline calendar/tests/test.bal:410]:
```suggestion
        log:printInfo("Event deleted with ID : " + eventId);
```
```suggestion
        log:printInfo("Event deleted with ID : " + eventId);
```

**Review by LakshanSS — APPROVED** [review]

File: calendar/tests/test.bal (hunk 1 above)
Comment (SkNuwanTissera) [reply calendar/tests/test.bal]: Formatted the code

File: calendar/tests/test.bal (hunk 3 above)
Comment (SkNuwanTissera) [reply calendar/tests/test.bal]: Formatted the code

File: calendar/tests/test.bal (hunk 2 above)
Comment (SkNuwanTissera) [reply calendar/tests/test.bal]: Formatted the code

**Pull request merged**